SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -283,6 +287,10 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	virtual int64 GetActorEntityId(const AActor& Actor) const override;
 
+	void AddSpatialShadowActor(const Worker_EntityId_Key EntityId);
```

Comment:
I think you can remove these form the NetDriver API and redirect those to SpatialNetDriverAuthorityDebugger internally.

Reply:
Done